src/features/Search/components/VisualNovelListOptions/data.ts

```diff
@@ -0,0 +1,42 @@
+import { VisualNovelListVariant } from '../VisualNovelList/VisualNovelList';
```

Comment:
I am opossing to idea to name this file as `data` since 1) it's too ambiguous and 2) actually there isn't any data here :)
Btw, what I meant by moving `buttonOptions` out of component isn't to move it to separate file, but to move it out of component scope, because  `buttonOptions` does not depend on any state of the component so it doesn't needed to be reinitilized every time on component rerender